common.gypi

```diff
@@ -80,7 +80,8 @@
     'conditions': [
       ['target_arch=="arm64"', {
         # Disabled pending https://github.com/nodejs/node/issues/23913.
```

Comment:
this comment and the new one below will need to change, probably removed entirely